Implementation/application/controllers/InfoMessage.php

```diff
@@ -34,6 +34,30 @@ public function PageNotFound(){
 
 		}
 
+		public function AuctionChanged(){
+			$id = $this->input->get("id");
+
+			$content["icon"] = "warning";
+			$content["message"] = "Auction has been changed in meantime!";
```

Comment:
Ovde samo dodaj jasniju poruku da je neko u medjuvremenu ponudio ponudu, npr "Someone has bidded in the meantime"
```suggestion
			$content["message"] = "Someone has bidded in the meantime!";
```